pkg/didcomm/protocol/route/support_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package route
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+	mockdispatcher "github.com/hyperledger/aries-framework-go/pkg/internal/mock/didcomm/dispatcher"
+	mockkms "github.com/hyperledger/aries-framework-go/pkg/internal/mock/kms"
+	mockstore "github.com/hyperledger/aries-framework-go/pkg/internal/mock/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/kms"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// mock route coordination provider
+type mockProvider struct {
+	openStoreErr error
+}
+
+func (p *mockProvider) OutboundDispatcher() dispatcher.Outbound {
+	return &mockdispatcher.MockOutbound{}
+}
+
+func (p *mockProvider) StorageProvider() storage.Provider {
+	if p.openStoreErr != nil {
+		return &mockstore.MockStoreProvider{ErrOpenStoreHandle: p.openStoreErr}
+	}
+
+	return mockstore.NewMockStoreProvider()
+}
+
+func (p *mockProvider) InboundTransportEndpoint() string {
+	return "ws//example.com"
```

Comment:
is this missing a :?

Reply:
[fixed](https://github.com/hyperledger/aries-framework-go/compare/d261ea92a76770ea17bc427f183b9cbda779bb62..f71d3c668644822b868c0ebec4387a83ea9d3b5d#diff-6f60e8f13e1dea1aa90429112324187fR36).